spec/features/desuscribir_spec.rb

```diff
@@ -9,17 +9,37 @@
     let!(:token) { suscripcion.token }
     let(:url) { "http://www.example.com/suscribir/resuscribir/#{ tipo }/#{ suscribible.id }/#{ email }/#{ token }" }
 
-    it 'muestra el enlace de desuscribir' do
+    it 'muestra el enlace de desuscribir, desuscribe y resuscribe' do
       visit "/suscribir/confirmar_baja/#{ suscripcion.id }/#{ suscripcion.token }"
       expect(page.status_code).to be(200)
       expect(page).to have_button('Sí, quiero cancelar mi suscripción')
+
       click_button 'Sí, quiero cancelar mi suscripción'
+      mail_enviado = ActionMailer::Base.deliveries.last
+      expect(mail_enviado).to be_present
+      expect(mail_enviado).to deliver_from(Suscribir::Personalizacion.email_contacto)
+      expect(mail_enviado).to deliver_to(suscripcion.email)
+      expect(mail_enviado).to have_body_text(/|#{url}|/)
       suscribible.reload
       expect(suscribible.suscripciones.count).to eq(0)
       expect(page).to have_link('activar de nuevo tu suscripción', href: url.gsub("\n", "%0A"))
+
       click_link 'activar de nuevo tu suscripción'
+      mail_enviado = ActionMailer::Base.deliveries.last
+      expect(mail_enviado).to be_present
+      expect(mail_enviado).to deliver_from(Suscribir::Personalizacion.email_contacto)
+      expect(mail_enviado).to deliver_to(suscripcion.email)
+      expect(mail_enviado).to have_subject('Suscripción confirmada')
+      suscribible.reload
+      expect(suscribible.suscripciones.count).to eq(1)
+      expect(page).to have_css('h1', 'Bienvenido de nuevo')
+
+      contador_enviados = ActionMailer::Base.deliveries.count
+      visit url.gsub("\n", "%0A")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.